contrib/clojure-package/src/org/apache/clojure_mxnet/io.clj

```diff
@@ -213,14 +245,24 @@
                last-batch-handle "pad"
                data-name "data"
                label-name "label"}}]
-   (new NDArrayIter
-        (util/vec->indexed-seq data)
-        (if label (util/vec->indexed-seq label) (util/empty-indexed-seq))
-        (int data-batch-size)
-        shuffle
-        last-batch-handle
-        data-name
-        label-name))
+   (let [specify-data-desc? (map? data)]
```

Comment:
nit: is this just in the `let` for a self-documenting name/rationale?

Reply:
I think it can be dropped - thanks